_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+{% auto_anchor %}
+
+- **What is a taproot?** Wikipedia [says][wikipedia taproot], "A taproot
+  is a large, central, and dominant root from which other roots sprout
+  laterally. Typically a taproot is somewhat straight and very thick, is
+  tapering in shape, and grows directly downward.  In some plants, such
+  as the carrot, the taproot is a storage organ so well developed that
+  it has been cultivated as a vegetable."
+
+    Bitcoin's [taproot][topic taproot], similar to the plant root, hides
+    the details of its scripts below the surface, away from public view,
+    and is very efficient at storing those scripts.
+
+- **Schnorr predates ECDSA:** we talk about [schnorr signatures][topic
```

Comment:
nit (also a few times below): I think I point this out at least once per month somewhere but I prefer to call this "Schnorr signatures" (or at least "Schnorr sigs") in formal writing. "Schnorr" is an alive human being and not a thing, so I think using his name without any suffixes for a signature scheme is a little bit disrepectful, and everyone deserves respect (even though he patented the scheme. ;)).